### ISSUE TITLE
ls -lR sometimes fails on OSX

### ISSUE THREAD

**Issue by kellyjonbrazil** [kellyjonbrazil/jc#40]
Need to investigate further.

Suspect it may be due to `total xx` line missing in empty folders:
```
./tmp:
total 0
drwxr-xr-x  2 kbrazil  staff   64 Mar  7 16:34 empty-folder
drwxr-xr-x  9 kbrazil  staff  288 Feb 27 10:19 lstest

./tmp/empty-folder:

./tmp/lstest:
total 0
-rw-r--r--  1 kbrazil  staff  0 Feb 27 10:19 ????this file starts with four newlines
-rw-r--r--  1 kbrazil  staff  0 Feb 27 10:19 ??this file has?a combination???of everything????
-rw-r--r--  1 kbrazil  staff  0 Feb 27 10:19 ?this file starts with one newline
-rw-r--r--  1 kbrazil  staff  0 Feb 27 10:19 a regular filename
-rw-r--r--  1 kbrazil  staff  0 Feb 27 10:19 this file?has?six??newlines??within
-rw-r--r--  1 kbrazil  staff  0 Feb 27 10:19 this file has????four contiguous newlines inside
-rw-r--r--  1 kbrazil  staff  0 Feb 27 10:19 this file has?a newline inside
```

**Comment by kellyjonbrazil** [comment]
Believe this is fixed in https://github.com/kellyjonbrazil/jc/commit/1e22f610a33903e8525d35a18b1dcf760a93791e

Need to add tests.

**Comment by kellyjonbrazil** [comment]
fix merged into 1.8.1
